src/main/java/com/bandwidth/sdk/messaging/models/Message.java

```diff
@@ -36,4 +37,14 @@ public boolean isMms(){
         return !isSms();
     }
 
+    @JsonIgnore
+    public Set<String> getReplyNumbers(){
+        Set<String> toNumbers = new TreeSet<>();
+        getTo()
+            .stream()
+            .filter(number -> !getOwner().equals(number))
+            .forEach(number -> toNumbers.add(number));
```

Comment:
You can use `.collect(...)` to put all the elements in a collection

Reply:
https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#collect-java.util.stream.Collector-